wavefront/requirements.yaml

```diff
@@ -1,5 +1,5 @@
 dependencies:
   - name: kube-state-metrics
     version: ^2.2.1
-    repository: https://kubernetes-charts.storage.googleapis.com/
+    repository: https://charts.helm.sh/stable
```

Comment:
I believe the helm stable repo has been deprecated a while back and no longer supported. 

Could we use `https://kubernetes.github.io/kube-state-metrics` instead? As specified under https://github.com/kubernetes/kube-state-metrics/tree/master/charts/kube-state-metrics

Reply:
@vikramraman that link is currently broken - https://github.com/kubernetes/kube-state-metrics/issues/1331

I think the best course of action would be to wait? Either way ` https://charts.helm.sh/stable` is the only working url at this time.

Reply:
Thanks. This works for now. We'll likely remove the kube-state part from our chart in the near future.